gspread/worksheet.py

```diff
@@ -464,7 +503,13 @@ def get_values(self, range_name=None, combine_merged_cells=False, **kwargs):
             worksheet.get_values('A2:B4', value_render_option=ValueRenderOption.formula)
         """
         try:
-            vals = fill_gaps(self.get(range_name, **kwargs))
+            if maintain_size:
```

Comment:
please compare it to a the very specific value you expect so here it should be `if maintain_size is True` in order to prevent false positive like if `maintain_size` is a string or anything else that evaluates to `True`.